### PR TITLE
fix: modernize build process, remove npm publish, use github actions

### PR DESCRIPTION
**Pull request by davidjoy**
This is best-reviewed commit-by-commit. 

This one PR does a few different pieces of maintenance on this repo in the interest of not dwelling too long on a low-priority MFE.  To wit: 

- It switches from travis to Github actions.
- It removes our attempt at publishing the MFE components to npm - we never used it.  (See commit for details)
- It fixes the old favicon this MFE still used.
- It fixes some incorrect environment variable defaults and an outdated doc URL.
- It removes a font we don't use anywhere else and which is just bloating the MFE.
-

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: ci
+on:
+  push:
+    branches:
+    - master
```
Comment (adamstankiewicz) [inline .github/workflows/ci.yml:6]: Should this only be run on `master`? Wouldn't we want this to run on push to any branch?

File: public/index.html [hunk 2]
```diff
@@ -4,9 +4,7 @@
       <title>Learner Profile | <%= process.env.SITE_NAME %></title>
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
-      <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
-      <link rel="shortcut icon" href="<%=webpackConfig.output.publicPath%>favicon.ico" type="image/x-icon" />
+      <link rel="shortcut icon" href="<%=htmlWebpackPlugin.options.FAVICON_URL%>" type="image/x-icon" />
```
Comment (adamstankiewicz) [inline public/index.html:7]: 🎉

File: .github/workflows/ci.yml (hunk 1 above)
Comment (davidjoy) [reply .github/workflows/ci.yml:6]: Hmm, well it all ran on _this_ branch... admittedly I cribbed this from frontend-app-account.

File: public/index.html (hunk 2 above)
Comment (davidjoy) [reply public/index.html:7]: I know, right?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (davidjoy) [reply .github/workflows/ci.yml:6]: Ah, it's the `pull_request` below that gives us what we want.  That causes it to run on PRs.

Comment (adamstankiewicz) [reply .github/workflows/ci.yml:6]: OH, sorry. The `pull_request` was already in there?! I missed that in the initial review, my bad!

**Pull request merged**